### PR TITLE
rename identifiers in method declarations

### PR DESCRIPTION
**Pull request by dariajung**
Method names re-named in Method Declarations

### CODE REVIEW

File: test/unit/main_test.ts [hunk 1]
```diff
@@ -79,7 +79,7 @@ describe('Recognizes invalid TypeScript inputs', () => {
 describe('Visitor pattern', () => {
   it('renames identifiers of property declarations and property access expressions', () => {
```
Comment (rkirov) [inline test/unit/main_test.ts]: change description. Probably will be too long so you can split to multiple tests.

**Comment by rkirov** [comment]
LGTM.

File: test/unit/main_test.ts (hunk 1 above)
Comment (dariajung) [inline test/unit/main_test.ts]: split

**Comment by dariajung** [comment]
merged

**Pull request closed without merging**